panel/widgets/slider.py

```diff
@@ -121,32 +124,73 @@ def _get_embed_state(self, root, values=None, max_opts=3):
         return (dw, w_model, values, lambda x: x.value, 'value', 'cb_obj.value')
 
 
-class FloatSlider(ContinuousSlider):
+class FloatSlider(_ContinuousSlider):
+    """The FloatSlider widget allows selecting a floating-point value within a
+    set of bounds using a slider.
 
-    start = param.Number(default=0.0)
+    Reference: https://panel.holoviz.org/reference/widgets/FloatSlider.html
 
-    end = param.Number(default=1.0)
+    ## Example
 
-    value = param.Number(default=0.0)
+    ```python
+    import panel as pn
 
-    value_throttled = param.Number(default=None, constant=True)
+    pn.extension()
```

Comment:
I think I wouldn't include the import and the call to `.extension()`. I've had a look at the pandas' docs and they don't include any import: https://pandas.pydata.org/docs/reference/api/pandas.merge.html

Reply:
I looked at the Streamlit documentation before hand. And they don't either. I chose to add it because then you get a working example. I will remove it to be in line with other libraries.

Reply:
Yes I agree with you there's pros and cons between the two approaches. Seems like the one without the imports is the preferred one around (the easier to maintain too I guess).